CMakeLists.txt

```diff
@@ -122,6 +122,7 @@ if(ENABLE_WEXTRA)
   set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wextra" )
 endif()
 
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
This is actually not as much of a change as it might look -- while gcc defaults to non-PIC, most (nearly all?) distros tweak their gcc to produce PIC by default (since it's needed for ASLR) including Ubuntu. Our pinned builds are all PIC.

So it most cases this is really a no-op change.

At one point clang didn't have the PIC tweak in Ubuntu but it appears they do it to clang now too.